pycm/curve.py

```diff
@@ -183,6 +188,23 @@ def __repr__(self) -> str:
         """Representation method."""
         return "pycm.ROCCurve(classes: " + str(self.classes) + ")"
 
+    def optimal_thresholds(self) -> Dict[Any, float]:
+        """
+        Get optimal thresholds for each class.
+
+        The optimal threshold is calculated based on "Closest to (0,1)" criterion (also known as the Euclidean distance
+        method or Youden's J statistic equivalent). The formula for calculating the distance is:
+        $optimal_cut_point = argmin_c √[(1-TPR(c))² + (FPR(c))²]$
+        """
+        optimal_thresholds = {}
+        for c in self.classes:
+            fpr = numpy.array(self.data[c][self.plot_x_axis])
+            tpr = numpy.array(self.data[c][self.plot_y_axis])
+            distances = numpy.sqrt(fpr ** 2 + (1 - tpr) ** 2).tolist()
```

Comment:
doesn't need to convert to list (remove `tolist()`)

Reply:
fixed in https://github.com/sepandhaghighi/pycm/pull/614/commits/5c79b032a22ffab77627d5c642b04cf988ec3316